subprojects/core/src/integTest/groovy/org/gradle/internal/operations/notify/BuildOperationNotificationIntegrationTest.groovy

```diff
@@ -168,10 +174,17 @@ class BuildOperationNotificationIntegrationTest extends AbstractIntegrationSpec
         succeeds "t"
 
         then:
-        started(EvaluateSettingsBuildOperationType.Details, [settingsDir: file('buildSrc').absolutePath, settingsFile: file('buildSrc/settings.gradle').absolutePath])
-        started(EvaluateSettingsBuildOperationType.Details, [settingsDir: file('a').absolutePath, settingsFile: file('a/settings.gradle').absolutePath])
-        started(EvaluateSettingsBuildOperationType.Details, [settingsDir: file('a/buildSrc').absolutePath, settingsFile: file('a/buildSrc/settings.gradle').absolutePath])
-        started(EvaluateSettingsBuildOperationType.Details, [settingsDir: file('.').absolutePath, settingsFile: file('settings.gradle').absolutePath])
+        started(LoadBuildBuildOperationType.Details, [buildPath:":"])
+
+        started(LoadBuildBuildOperationType.Details, [buildPath:":"])
```

Comment:
Given that now, this is the only way we can test the hierarchy for composites, we need to do that here. 
This test needs to assert what the tree looks for composites (i.e. what is a child of what).

Reply:
reworked the test and added hierarchy checks